### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added the CC MD guide.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
I think we should also add that you should not include words like `you`, `your` `our` etc in entries but I'm not sure where it should go.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -109,6 +109,7 @@ To link your Codecademy user profile to GitHub:
 - We require at least two round of reviews from the [content team members](https://github.com/codecademy/docs#-content-team). Make sure to make the changes after each round.
 - Requested changes must be resolved before merging.
 - Your entry will be deployed within the hour after it's merged!
+- Read the [Codecademy Markdown Guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).
```
Comment (Dusch4593) [inline .github/CONTRIBUTING.md:112]:
```suggestion
- Refer to the [Codecademy Markdown Style Guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).
```

**Comment by ethan-master-coding** [comment]
@Dusch4593 Made the chance.

**Comment by Dusch4593** [comment]
> I think we should also add that you should not include words like `you`, `your` `our` etc in entries but I'm not sure where it should go.

This is actually already covered in the [content-standards.md file](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md#general-writing-tips) under "General Writing Tips"

**Review by Dusch4593 — APPROVED** [review]

**Comment by Dusch4593** [comment]
Looks like the Markdown style guide is linked in `content-standards.md` as well 😄

**Pull request merged**